content/guides/tagging.md

```diff
@@ -19,7 +19,7 @@ The following integration sources create tags automatically in Datadog:
 | Amazon Web Services | Region, Account, Hostname,  Org ID,  AZ, Kernel, Image, Instance Type, Security Group,   Tags assigned in console or via AWS tools |
 | AWS SQS | Queue Name |
 | Apache | Apache Host and Port |
-| Azure | Tenant Name, Status, Tags, Subscription ID and Name |
+| Azure | Tenant Name, Status, Tags, Subscription ID and Name, Availability Zone in common with AWS tag after contacting Datadog support |
```

Comment:
Azure, Google, and AWS seemed to use different names for these. I believe its

Availability-zone, zone, and availability_zone respectively.

We should confirm. There was a card in the eclair backlog to unify them.  But not sure if its complete.